sql_generators/active_users_aggregates_v3/templates/mobile_query.sql

```diff
@@ -61,75 +80,74 @@ search_clients AS (
 ),
 search_metrics AS (
   SELECT
-    baseline.client_id,
-    baseline.submission_date,
+    unioned.client_id,
+    unioned.submission_date,
     SUM(ad_click) AS ad_clicks,
     SUM(organic) AS organic_search_count,
     SUM(search_count) AS search_count,
     SUM(search_with_ads) AS search_with_ads
   FROM
-    baseline
+    unioned
   LEFT JOIN
     search_clients s
-    ON baseline.client_id = s.client_id
-    AND baseline.submission_date = s.submission_date
+    ON unioned.client_id = s.client_id
+    AND unioned.submission_date = s.submission_date
   GROUP BY
     client_id,
     submission_date
 ),
-baseline_with_searches AS (
+unioned_with_searches AS (
   SELECT
-    baseline.client_id,
+    unioned.client_id,
     CASE
-      WHEN BIT_COUNT(days_seen_bits)
+      WHEN BIT_COUNT(days_active_bits)
         BETWEEN 1
         AND 6
         THEN 'infrequent_user'
-      WHEN BIT_COUNT(days_seen_bits)
+      WHEN BIT_COUNT(days_active_bits)
         BETWEEN 7
         AND 13
         THEN 'casual_user'
-      WHEN BIT_COUNT(days_seen_bits)
+      WHEN BIT_COUNT(days_active_bits)
         BETWEEN 14
         AND 20
         THEN 'regular_user'
-      WHEN BIT_COUNT(days_seen_bits) >= 21
+      WHEN BIT_COUNT(days_active_bits) >= 21
         THEN 'core_user'
       ELSE 'other'
     END AS activity_segment,
```

Comment:
Are these changes (using `days_active_bits` in place of `days_seen_bits`) proposed or documented somewhere? I didn't realize we talked about changing the calculation for these activity segments.

Reply:
Only informally in a [slack thread](https://mozilla.slack.com/archives/C06LZ00TARL/p1713470529167759). Do you prefer creating a Jira task to be approved?